README.md

```diff
@@ -59,29 +59,50 @@ Replace `<service name> with the desired name of the service`.
 
 ### Tracer activation
 
-There are two ways to activate the `lumigo_opentelemetry` package: one based on importing the package in code (manual activation), and the other via the environment (no-code activation).
-The [no-code activation](#no-code-activation) approach is the preferred one.
+There are several ways to activate the `lumigo_opentelemetry` package:
+
+1. [No-code activation](#no-code-activation) (preferred approach).
+2. [Manual activation](#manual-activation).
+3. [Decorator-based activation](#decorator-based-activation).
 
 #### No-code activation
 
 **Note:** The instructions in this section are mutually exclusive with those provided in the [Manual activation](#manual-activation) section.
 
 Set the following environment variable:
 
-```console
+```sh
 AUTOWRAPT_BOOTSTRAP=lumigo_opentelemetry
 ```
 
 #### Manual activation
 
-**Note:** The instructions in this section are mutually exclusive with those provided in the [No-code activation](#no-code-activation) section.
+**Note:** The instructions in this section are mutually exclusive with those provided in the [No-code activation](#no-code-activation) and [Decorator-based activation](#decorator-based-activation) sections.
 
 Import `lumigo_opentelemetry` at the beginning of your main file:
 
 ```python
 import lumigo_opentelemetry
 ```
 
+### Decorator-based activation
+
+For simple Python scripts that are not built around Lumigo or OpenTelemetry-instrumented libraries, you can use the `@lumigo_wrapped` decorator for activation. This approach is especially useful for standalone scripts, cron jobs, or similar use cases.
```

Comment:
You need to first mention that by default if using the autowrapping or manual activation then the script will be instrumented, but there are limitations such as:
* The last spans might be lost if the script ends before they are sent
* If no outbound requests are made (For example if the code crashes before the first call is made) then the script run wont generate any traces in lumigo